web/src/actions/chatActions.js

```diff
@@ -0,0 +1,37 @@
+import { ADD_MESSAGE, MESSAGE_RECEIVED, ADD_USER, USERS_LIST } from './types';
+
+let nextMessageId = 0;
+let nextUserId = 0;
+
+export const addMessage = (message, author) => {
+  return {
+    type: ADD_MESSAGE,
+    id: nextMessageId++,
```

Comment:
this is okay cause it's temporary code, but fyi you should not make any changes inside of actions, they should happen before you create the action